### PR TITLE
Migrate to lucide-vue for icons and update copying info

### PR DESCRIPTION
**Pull request by triphora**

### CODE REVIEW

**Comment by Prospector** [comment]
I started making a list of icons that I felt looked notably worse than before, but I'm not sure it's worth actually fixing these because just using all lucide would really be quite helpful. The only one I think definitely needs to be changed is the money icon, it's just not recognizable

Note | Current |  Lucide
:-:|-|:-:
Download icon just looks weird and poorly proportioned | image  |  image
A bill at this size just looks like a rectangle and not money | image | image
The narrower shoulders on the Account is just less distinct and is just marginally worse | image | image
Project cards use an "Edit" icon instead of an "Updated" icon | image | image
Search icon looks unbalanced and poorly proportioned | image | image
New caret doesn't match the font weight as nicely | image | image

File: pages/dashboard.vue [hunk 1]
```diff
@@ -30,14 +30,16 @@
 </template>
 
 <script>
+import {
+  LayoutDashboard as DashboardIcon,
+  // BarChart3 as ChartIcon,
+  Banknote as CurrencyIcon,
```
Comment (Prospector) [inline pages/dashboard.vue:36]: Use dollar sign instead of bank note

File: pages/settings.vue [hunk 2]
```diff
@@ -33,14 +33,16 @@
 </template>
 
 <script>
+import {
+  Paintbrush as PaintbrushIcon,
+  User as UserIcon,
+  Heart as HeartIcon,
+  Banknote as CurrencyIcon,
```
Comment (Prospector) [inline pages/settings.vue:40]: Use dollar sign instead of bank note

**Review by Prospector — CHANGES_REQUESTED** [review]

**Comment by Prospector** [comment]
I actually just noticed this one, without the circle it doesn't even look like the GitHub logo:

image

**Comment by triphora** [comment]
Closing in favour of #760

**Pull request closed without merging**